DEVDOCS-3013: [update] GraphQL carts & checkout (#1940)
bigcommerce/dev-docs: docs/api-docs/getting-started/about-our-api.mdx

### GraphQL Storefront API
BigCommerce's [GraphQL Storefront API](/docs/graphql-storefront) allows you to query products, customers, carts and launch a checkout on a headless storefront or from a store's frontend. Example use cases include the following:

> * Request product data from your browser into a Stencil theme

Reviewer: ```suggestion
* Add additional product data to a Stencil storefront
```

* Access customer data on the frontend of a site

> * Fetch category and brand details from a store's frontend

Reviewer: ```suggestion
* Manage shopper's carts on a headless storefront
* Fetch category and brand details from a store's frontend
```
